### PR TITLE
[Asset] CLI: use request context to generate absolute URLs

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #15448 |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

File: src/Symfony/Bridge/Twig/Extension/HttpFoundationExtension.php [hunk 1]
```diff
@@ -12,7 +12,7 @@
 namespace Symfony\Bridge\Twig\Extension;
 
 use Symfony\Component\HttpFoundation\RequestStack;
-use Symfony\Component\HttpFoundation\Request;
```
Comment (fabpot) [inline src/Symfony/Bridge/Twig/Extension/HttpFoundationExtension.php]: This should be kept (there is a reference in a @see annotation)

**Comment by fabpot** [comment]
:+1:

File: src/Symfony/Bridge/Twig/Extension/HttpFoundationExtension.php (hunk 1 above)
Comment (xabbuh) [inline src/Symfony/Bridge/Twig/Extension/HttpFoundationExtension.php]: Good catch, PhpStorm doesn't seem to handle that properly. I changed the annotation to use the FQCN instead. Okay for you?

Comment (fabpot) [inline src/Symfony/Bridge/Twig/Extension/HttpFoundationExtension.php]: No, we are using short class names everywhere else.

**Comment by javiereguiluz** [comment]
@xabbuh a very nice solution for a very demanded feature. Thanks!

File: src/Symfony/Bridge/Twig/Extension/HttpFoundationExtension.php (hunk 1 above)
Comment (xabbuh) [inline src/Symfony/Bridge/Twig/Extension/HttpFoundationExtension.php]: okay, I reverted this one

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**